class DataFuturaException(Exception):
    def __init__(self):
        self.mensagem = (
            "Data n√£o pode ser no futuro."
        )
        super().__init__(self.mensagem)
